CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing Guidelines
+
+## New Release
```

Comment:
I find this kind of information extremely useful for maintainers, but still don't know if we should store it in the `CONTRIBUTING.md` file. Anyhow, it's better to have it here than not having it at all.

Reply:
I haven't seen any other better places to put it (apart from perhaps [wiki](https://github.com/iterative/cml/wiki))?

Reply:
...Which also seems to have a healthy dose of redundant information that users can also find on the `README.md` file, the dvc.org/doc/cml page and under the cml.dev domain. This is starting to look like [a cumuative song](https://en.wikipedia.org/wiki/Cumulative_song). 😅 

I usually see this kind of information in a separate `MAINTAINERS.md` file (random search results: [1](https://github.com/zendframework/maintainers/blob/master/MAINTAINERS.md), [2](https://github.com/mochajs/mocha/blob/master/MAINTAINERS.md), [3](https://gitlab.gnome.org/GNOME/gnome-control-center/blob/master/docs/MAINTAINERS.md)) along with the list of maintainers, while `CONTRIBUTING.md` is more oriented towards casual/external contributors.

The wiki could be another good place to store this kind of information, but might be less portable than a standalone file.

Reply:
It takes time to understand people and the motivations behind their most quotidian decisions.
https://github.com/tqdm/tqdm/blob/master/CONTRIBUTING.md#manage-a-new-release

You convinced me: with such a careful layout, `CONTRIBUTING.md` doesn't necessarily have to be only for external contributions.